src/status_im/data_store/realm/schemas/account/core.cljs

```diff
@@ -1,10 +1,10 @@
 (ns status-im.data-store.realm.schemas.account.core
-  (:require [status-im.data-store.realm.schemas.account.v1.core :as v1]))
+  (:require [status-im.data-store.realm.schemas.account.v2.core :as v2]))
 
 ;; TODO(oskarth): Add failing test if directory vXX exists but isn't in schemas.
 
 ;; put schemas ordered by version
-(def schemas [{:schema        v1/schema
+(def schemas [{:schema        v2/schema
                :schemaVersion 1
```

Comment:
Complete newbie comment here, but is this `schemaVersion` supposed to stay at 1?

Reply:
no, correct, needs to be bumped to version 2, I have the change locally, but haven't push